packages/mobile/src/screens/settings-screen/CommentSettingsScreen.tsx

```diff
@@ -0,0 +1,166 @@
+import { useCallback } from 'react'
+
+import { useGetCurrentUserId, useGetMutedUsers } from '@audius/common/api'
+import { useMuteUser } from '@audius/common/context'
+import { useSelectTierInfo } from '@audius/common/hooks'
+import { Status } from '@audius/common/models'
+import { formatCount } from '@audius/common/utils'
+import { TouchableWithoutFeedback } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import { useToggle } from 'react-use'
+
+import {
+  Flex,
+  Text,
+  Button,
+  Divider,
+  IconMessageBlock,
+  IconUser,
+  IconVerified
+} from '@audius/harmony-native'
+import { IconAudioBadge } from 'app/components/audio-rewards'
+import { Screen, ScreenContent, ProfilePicture } from 'app/components/core'
+import { LoadingSpinner } from 'app/harmony-native/components/LoadingSpinner/LoadingSpinner'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { makeStyles } from 'app/styles'
+import { useThemePalette } from 'app/utils/theme'
+
+const messages = {
```

Comment:
Lets put these messages in common and share across mobile/web!
I imagine these are identical to the ones used on web, so we can put it into `common/src/messages/comments` (feel free to organize in there how you see fit)